package.json

```diff
@@ -95,17 +102,20 @@
     "grunt-parallel": "^0.5.1",
     "html-entities": "^1.2.0",
     "husky": "^1.1.1",
+    "inquirer": "^6.2.0",
     "jquery": "^3.0.0",
     "jsdoc": "^3.5.5",
     "lint-staged": "^8.0.0",
     "markdown-table": "^1.1.2",
     "minami": "^1.2.3",
     "mocha": "^5.2.0",
+    "node-command-line": "^1.0.2",
```

Comment:
Consider replacing this with [`execa`](https://www.npmjs.com/package/execa). It's *much* better supported/trusted, and handles tons of weird Windows edge cases.